doc/org.eclipse.python4capella.doc/doc/user/general/Debug.textile

```diff
@@ -0,0 +1,28 @@
+h1. Debug a Pyhton script
+
+This section explains how you can debug your Python4Capella script.
+
+
+h2. Within Capella
+
+You will need to enable Development capabilities if it's not already done:
```

Comment:
You should create a section "Enable Development Capabilities" en just refers to this section here.